CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     if(MSVC)
         add_definitions("/W4")
     else()
-        add_definitions(-Wall -Wextra -pedantic)
+        add_definitions(-Wall -Wextra -pedantic -Wpessimizing-move)
```

Comment:
This new warning is not available on GCC, at least in MSYS2,  causes the compilation to fail, and isn't -Wpessimizing-move part of -Wall?    The failure is possibly due to some of gtest being C code and that isn't a valid warning in C.  

I had to remove it to get things to compile on MSYS2,  (not CLI11) itself just the test suite and everything.

Reply:
Sorry, didn't mean to commit it! It's added by the other flags anyway, I think.